README.md

```diff
@@ -1,12 +1,18 @@
 # Epic Games
 
+<div align="center">
+  <img src="https://cdn2.unrealengine.com/Epic+Games+Node%2Fxlarge_whitetext_blackback_epiclogo_504x512_1529964470588-503x512-ac795e81c54b27aaa2e196456dd307bfe4ca3ca4.jpg" width="20%" min-width="100px" />
+</div>
+  
 Unreal Engine is now [free](https://www.unrealengine.com/blog/ue4-is-free)!
 
-To access our repositories, sign up for a free account at [UnrealEngine.com](https://www.unrealengine.com) and register your GitHub ID using [these instructions](https://www.unrealengine.com/ue4-on-github). 
+To gain access for our repositories, sign up a free account on [UnrealEngine.com](https://www.unrealengine.com) and register your GitHub ID using [these instructions](https://www.unrealengine.com/ue4-on-github). 
 
-After that, you can find our repositories here:
+After that, you may able to find our repositories here:
 
 *  [Unreal Engine](https://github.com/EpicGames/UnrealEngine)
 *  [Unreal Tournament](https://github.com/EpicGames/UnrealTournament)
   
 (Note that you must be signed into GitHub for these links to work.)
+
+Have Fun !!
```

Comment:
@keplersj is correct, this should be changed to be less threatening:

```suggestion
May Able To Have Fun ?
```

Reply:
I don't like micro-aggressions with exclamation marks either, seems better now as a rhetorical question.